nemo/collections/asr/parts/submodules/rnnt_greedy_decoding.py

```diff
@@ -2202,3 +2494,4 @@ class GreedyBatchedRNNTInferConfig:
     preserve_alignments: bool = False
     preserve_frame_confidence: bool = False
     confidence_method_cfg: Optional[ConfidenceMethodConfig] = None
+    cuda_optimized: bool = True
```

Comment:
Are the results exactly equivalent to greedy batched? If so this is fine, otherwise use default as false